src/cart/crt_hg.c

```diff
@@ -1154,9 +1154,9 @@ crt_rpc_handler_common(hg_handle_t hg_hdl)
 	else
 		rc = crt_corpc_common_hdlr(rpc_priv);
 	if (unlikely(rc != 0)) {
-		RPC_ERROR(rpc_priv,
-			  "failed to invoke RPC handler, rc: "DF_RC"\n",
-			  DP_RC(rc));
+		RPC_WARN(rpc_priv,
+			 "failed to invoke RPC handler, rc: "DF_RC"\n",
+			 DP_RC(rc));
```

Comment:
i expect either crp_rpc_common_hdlr() or crt_corpc_common_hdlr() to print reason for failure if they return rc != 0, so probably we can just nuke message here altogether

Reply:
Yes, I kept the error message over there. How about we change this to INFO just in case it will miss thing?